lm_eval/models/mlc_llm.py

```diff
@@ -298,6 +298,7 @@ def create_chat_completion_payload(
             ],
             "stream": False,
             "stop": stop_tokens,
+            "max_tokens": self.max_gen_toks,
```

Comment:
There is no class field `max_gen_toks`, it shoud be defined in the class and control by high-level API